config/initializers/sentry.rb

```diff
@@ -0,0 +1,48 @@
+require "sentry-raven"
+
+Raven.configure do |config|
+  config.dsn = ENV["SENTRY_DSN"]
+  config.environments = %w[staging production]
+  config.current_environment = Rails.env.to_s
```

Comment:
Can't we declare the `current_environment` (probably from an env variable) without previously listing every available environment?

I'd like to setup a new instance (with a new environment) without having to update every other instance out there to reflect the change.

I'd rather do `config.current_environment = ENV['SENTRY_CURRENT_ENVIRONMENT']` (probably with a `.present?`) than these two lines, but I'm not sure if the `.environments` is required.

Reply:
The `environments` array is here so we only report in the  `development`. It could be omitted so it'd only report when SENTRY_DSN is set.